packages/terra-clinical-detail-view/src/DetailList.jsx

```diff
@@ -21,27 +21,53 @@ const propTypes = {
     PropTypes.objectOf(DetailListItem),
     PropTypes.arrayOf(PropTypes.objectOf(DetailListItem)),
   ]),
+
+  /**
+   * The list type based on whether the list items are single HTML elements or label-value pairs.
+   * Label-value pairs should be provided using the `<dt>` element for the label and the `<dd>` element for the value.
+   * The `terra-clinical-label-value-view` component with `isChildOfDescriptionList={true}` can also be used as it will
+   * return the label and value using the appropriate `<dt>` and `<dd>` elements.
+   * One of `'singleItem'` or `'labelValuePair'`.
+   */
+  type: PropTypes.oneOf(['singleItem', 'labelValuePair']),
 };
 
 const defaultProps = {
   title: undefined,
   children: undefined,
+  type: 'singleItem',
```

Comment:
Could this 'singleItem' be confused with 1 item in the whole list? Wondering if there is a better name for it.

Reply:
Open to suggestions

Reply:
I've been brainstorming ideas for this but I haven't come up with anything that may not also be confusing for consumers. I'm starting to lean towards a boolean prop for this again as I think it makes since for a DetailList to just be an unordered list by default. Maybe we just use a name that doesn't reference the implementation, something like `isLabelValuePairList`?

Reply:
Boolean prop makes more sense to me as well instead of an enum.

Reply:
+1

Reply:
Updated here: 452d0512acab93f891e9c99203abf3f7a0ec0138